x/bridge/types/params_test.go

```diff
@@ -0,0 +1,147 @@
+package types_test
+
+import (
+	"testing"
+
+	math "cosmossdk.io/math"
+	"github.com/cometbft/cometbft/crypto/ed25519"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+// TestParams tests if Params are properly validated.
+func TestParams(t *testing.T) {
+	var (
+		pk1        = ed25519.GenPrivKey().PubKey()
+		addr1Bytes = sdk.AccAddress(pk1.Address())
+		addr1      = addr1Bytes.String()
+
+		assetID1 = types.AssetID{
+			SourceChain: "bitcoin",
+			Denom:       "wbtc1",
+		}
+		assetID2 = types.AssetID{
+			SourceChain: "bitcoin",
+			Denom:       "wbtc2",
+		}
+	)
```

Comment:
Yeah I think we just define this top level.

Also, consider moving over to a KeeperTestSuite instead of testing.T. It might be more helpful once we start making more complex test cases, especially will help with reducing replication.

Reply:
Moved everything to `helpers_test.go`